### DISCUSSION TITLE
Couldn't find a debug adapter descriptor for debug type 'python'

### DISCUSSION THREAD

**Discussion by DFofanov** [coder/code-server#2369] [Q&A]
I encountered the following issue while trying to debug Python code in coder-server.
![image](https://user-images.githubusercontent.com/9949610/100393349-63805500-304a-11eb-97e5-abacbcf2f076.png)

Below is my environment information.
Web Browser: Chrome 86
Local OS: macOS Catalina
Remote OS: Ubuntu LTS
Remote Architecture:
code-server --version: 3.6.7

And my launch.json
![image](https://user-images.githubusercontent.com/9949610/100393449-b8bc6680-304a-11eb-8530-3d602a3ce9ff.png)

![image](https://user-images.githubusercontent.com/9949610/100393410-99253e00-304a-11eb-801a-c6aa2a5e365b.png)

**Comment by code-asher** [comment]
I'm not 100% sure but I think this might be fixed now. Could you try out
3.7.3?
